WordPress/Classes/ViewRelated/Media/MediaLibraryViewController.swift

```diff
@@ -524,11 +528,16 @@ class MediaLibraryViewController: WPMediaPickerViewController {
     // MARK: - Upload Media from Camera
 
     private func presentMediaCapture() {
+        if FeatureFlag.asyncUploadsInMediaLibrary.enabled {
+            self.useUploadCoordinator = true
+        }
+
         capturePresenter = WPMediaCapturePresenter(presenting: self)
         capturePresenter!.completionBlock = { [weak self] mediaInfo in
             if let mediaInfo = mediaInfo as NSDictionary? {
                 self?.processMediaCaptured(mediaInfo)
             }
+            self?.useUploadCoordinator = false
```

Comment:
This should only be changed to false after the lines of 559 is executed or else because the processMediaCaptured is async when the check is done for the variable the value can be false.